ProfilerGUI/Source/Configurator/WindowsShortcutReader.cs

```diff
@@ -0,0 +1,56 @@
+﻿using IWshRuntimeLibrary;
+
+namespace ProfilerGUI.Source.Configurator
+{
+    /// <summary>
+    /// Parses Windows .lnk files.
+    /// </summary>
+    public class WindowsShortcutReader
+    {
+        /// <summary>
+        /// The extension of shortcut files on Windows.
+        /// </summary>
+        public const string ShortcutExtension = ".lnk";
+
+        /// <summary>
+        /// Parses the given .lnk file.
+        /// </summary>
+        public static ShortcutInfo ReadShortcutInfo(string filePath)
+        {
+            WshShell shell = new WshShell();
+            IWshShortcut link = (IWshShortcut)shell.CreateShortcut(filePath);
+            return new ShortcutInfo(link.TargetPath, link.WorkingDirectory, link.Arguments);
+        }
+
+        /// <summary>
+        /// Result of parsing a .lnk file.
+        /// </summary>
+        public class ShortcutInfo
```

Comment:
ditch helper class shortcutreader and add constructor/factory method ShortcutInfo.Read()

Reply:
...and ShortcutInfo is a really bad name. why not simply Shurtcut?

Reply:
done